dialogs/graph_settings.py

```diff
@@ -84,16 +99,20 @@ def accept_settings(self):
         self.parent.transpose_axes = \
             self.transposeAxesCheckBox.checkState() == QtCore.Qt.Checked
         self.parent.color_scheme = self.color_scheme
-        if self.radio_range_auto.isChecked():
-            self.parent.axes_range_mode = 0
-        elif self.radio_range_manual.isChecked():
+        
+        if self.radio_range_auto.isChecked(): 
+            self.parent.axes_range_mode = 0    
+                                                        
+         
+        elif self.radio_range_manual.isChecked():       
```

Comment:
```suggestion
        if self.radio_range_auto.isChecked(): 
            self.parent.axes_range_mode = 0    

        elif self.radio_range_manual.isChecked():
```
Code style issue:
- too many blank lines (2)